Add kwarg for angle style

rmatsum836 opened the pull request:
Addition of `angle_style` kwarg to specify if angles should be `fixed` or `harmonic`.  Right now, I'm trying to figure out how to test this feature.  I initially added this as a kwarg for some of the tests with the methane molecule, but I don't think OPLS methane is supposed to have fixed angles.  As a result, the simulation fails.

rmatsum836 commented:
The NVT example now runs a simulation of a water system now rather than a methane.  The reason is so that the `angle_style` kwarg could get tested.  

A FF xml for SPCE is contained in `ff_files` and the path of any xml contained in that directory can be grabbed from `utils.get_ff.get_ff_path()`.  

I would like to clean this up a little more and add a couple more tests before serious review.

rsdefever commented:
> I would like to clean this up a little more and add a couple more tests before serious review.

Can we get this ready to go in the next couple of days?

rmatsum836 commented:
Should be ready for review now.  In my most recent commit I added a conditional in `write_mcfs` that checks that the `angle_style` is valid.  I also added several tests in `test_writers.py`.
